packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -28,6 +28,7 @@ export interface CommerceApiProviderProps extends ApiClientConfigParams {
     locale: string
     currency: string
     queryClientConfig?: QueryClientConfig
+    correlationId?: string
```

Comment:
Should we make correlation required for the provider?

Reply:
Leave it as optional. If the API worked without having a correlation id sent to it, it implies that it's optional.